deepchecks/base/dataset.py

```diff
@@ -62,12 +62,13 @@ class Dataset:
     def __init__(
         self,
         df: pd.DataFrame,
+        label: pd.Series = None,
```

Comment:
why

Reply:
I wrote in the PR description, to allow the labels to be passed separated from data. like we do with `from_numpy`